Makefile

```diff
@@ -161,6 +161,7 @@ install.cni.sudo: gopath
 .PHONY: install
 install:
 	install -d -m 755 $(DESTDIR)/$(BINDIR)
+	test -z "${SELINUXOPT}" || chcon --verbose --reference=$(DESTDIR)$(BINDIR)/podman bin/buildah
```

Comment:
SELINUXOPT is not defined in the buildah makefile so you also have to copy over
`SELINUXOPT ?= $(shell test -x /usr/sbin/selinuxenabled && selinuxenabled && echo -Z)`